man/org.freedesktop.systemd1.xml

```diff
@@ -277,6 +277,8 @@ node /org/freedesktop/systemd1 {
       GetDynamicUsers(out a(us) users);
       DumpUnitFileDescriptorStore(in  s name,
                                   out a(suuutuusu) entries);
+      @org.freedesktop.systemd1.Privileged("true")
+      NFTSetReload();
```

Comment:
hmm, i find this awfully specific...

Shouldn't this just be a side-effect of the usual daemon reload operation?

Reply:
Maybe. This is faster though, reload takes some time.

Reply:
done